### PR TITLE
Refactoring BridgeConstants Class

### PR DESCRIPTION
**Pull request by wilmerrootstock**
Refactoring BridgeConstants Class.

## Description
- Moving BridgeConstants class and all classes (regtest, testnet, mainnet, devnet) related to `co.rsk.peg.constants` package to `Peg` package.
- Moving the BridgeConstantsTest class to the same package inside the test section.

## Motivation and Context

We are passing all the objects to `Peg` package to put all objects related to Powpeg together. With this, we are taking a step forward to decouple Powpeg.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Requires Activation Code (Hard Fork)

* **Other information**:

### CODE REVIEW

**Review by nathanieliov — APPROVED** [review]
LGTM

File: rskj-core/src/test/java/co/rsk/peg/constants/BridgeConstantsTest.java [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
 import co.rsk.bitcoinj.core.Coin;
 import java.util.stream.Stream;
+
+import co.rsk.peg.constants.BridgeConstants;
+import co.rsk.peg.constants.BridgeMainNetConstants;
+import co.rsk.peg.constants.BridgeRegTestConstants;
+import co.rsk.peg.constants.BridgeTestNetConstants;
```
Comment (marcos-iov) [inline rskj-core/src/test/java/co/rsk/peg/constants/BridgeConstantsTest.java:13]: Is this required?

**Review by marcos-iov — CHANGES_REQUESTED** [review]
Nice work Wilmer, well done.

Left a comment, I think there might be some unused imports added. Also check the report from SonarCloud in the PR, it shows some code smells

**Comment by apancorb** [comment]
Looking at the code smell, this should be fixed using `List.of()` which would make the list immutable. Since the project seems to be using Java 8, the same could be achieved using `Collections.unmodifiableList(Arrays.asList(...))`. I feel like this would be simpler than adding a wrapper method to access the list. Wdyt?

File: rskj-core/src/test/java/co/rsk/peg/constants/BridgeConstantsTest.java (hunk 1 above)
Comment (wilmerrootstock) [reply rskj-core/src/test/java/co/rsk/peg/constants/BridgeConstantsTest.java]: I just optimized the imports in a new commit.

**Comment by wilmerrootstock** [comment]
> Looking at the code smell, this should be fixed using `List.of()` which would make the list immutable. Since the project seems to be using Java 8, the same could be achieved using `Collections.unmodifiableList(Arrays.asList(...))`. I feel like this would be simpler than adding a wrapper method to access the list. Wdyt?

@apancorb @marcos-iov  Done!

**Review by marcos-iov — APPROVED** [review]

**Review by apancorb — APPROVED** [review]

**Pull request merged**